.github/workflows/publish-apispec.yaml

```diff
@@ -0,0 +1,135 @@
+name: '🚀 Publish OpenAPI Spec'
+
+on:
+  push:
+    branches:
+      - master
+      - main
```

Comment:
Why the main branch in all files? We use `master` in all our repositories.

Reply:
yes, you are 100% right. I simply did that out of convention xD working with both in other areas. If it bugs you and the rest of the server crew I can remove `main` during the final refactor before moving out of draft.